Core/Utilities/ElasticSearchDB.py

```diff
@@ -15,9 +17,51 @@
 from DIRAC import gLogger, S_OK, S_ERROR
 from DIRAC.Core.Utilities import Time
 from DIRAC.Core.Utilities import DErrno
+from DIRAC.Core.Security import Locations, X509Chain
 
 __RCSID__ = "$Id$"
 
+def getCert():
+  """
+  get the host certificate
+  """
+  cert = Locations.getHostCertificateAndKeyLocation()
+  if cert:
+    cert = cert[0]
+  else:
+    cert = "/opt/dirac/etc/grid-security/hostcert.pem"
+  return cert
+
+def generateCAFile():
```

Comment:
BTW it seems for the elasticsearch PR you need at least to sign a contributor's agreement for the PR to be merged.